vaadin-card-flow-parent/vaadin-card-flow/src/main/java/com/vaadin/flow/component/card/Card.java

```diff
@@ -100,11 +163,25 @@ public void setTitle(Component title) {
     }
 
     /**
-     * Gets the current title component.
+     * Gets the value of the {@code cardTitle} property. Returns empty if no
+     * title is set.
+     *
+     * @return the value of the title property
+     */
+    @Synchronize(property = "cardTitle", value = "card-title-changed")
```

Comment:
Why do we need to synchronize this from the client?

Reply:
It was a leftover, removed.